### ISSUE TITLE
update the wiki about the transformer

### ISSUE THREAD

**Issue by Grummfy** [dingo/api#1437]
Hello,
I think adding some word about how to create a transformer (like extending \League\Fractal\TransformerAbstract or read https://fractal.thephpleague.com/transformers/ for more information) could be really helpfull and make people win a lot of time

**Comment by catalinux** [comment]
I think it says: https://github.com/dingo/api/wiki/Transformers 

> Fractal
> 
> Fractal is the default transformation layer used by the package. It boasts a number of useful features > to help you keep your data consistent.

> To use Fractal it's recommend you read through the documentation found on their website.

**Comment by Grummfy** [comment]
Yes, but having a bit more information, doesn't cost that much. Saying which class to extends.... Having a little help, make a lot of difference

**Comment by kirkbushell** [comment]
@Grummfy I agree. nearly 3 years later, encountered the same issue - I'm actually stuck setting up transformers as I'm seeing errors but I've followed the wiki docs. Little bit frustrating.

**Comment by specialtactics** [comment]
I'm open to suggestions on things to add, but there's already a lot of info there.
